README.md

```diff
@@ -1242,6 +1244,125 @@ spec:
     emphemeral: true # VERY important. otherwise data inside the workdir and /tmp is not cleared between builds
 ```
 
+#### Docker image layers caching
+
+> **Note**: Ensure that the volume mount is added to the container that is running the Docker daemon.
```

Comment:
GitHub MD seems to add a little icon before `**Note**` only when it's surrounded with `**`. Perhaps it's a bit more appealing with that? 😄 

https://github.com/actions-runner-controller/actions-runner-controller/blob/pv-doc/README.md#docker-image-layers-caching